up-l1/zenoh.adoc

```diff
@@ -98,22 +108,21 @@ Different message types **MUST** use different Zenoh API.
 
 === URI Mapping
 
-UUri **MUST** be able to be serialized into link:../basics/uri.adoc#42-micro-uris[micro form].
-Then we can transform the micro-form UUri, which is `u8` array, into string in lowercase hexadecimal representation.
+UUri **MUST** be able to be serialized into link:../basics/uri.adoc#short-uris[short form].
+Then we can transform the short-form UUri into string for Zenoh key.
 
 Zenoh key for uProtocol consists of three parts and they are separated by `/`.
-Zenoh key **MUST** start with `upl/` or `upr/`.
-While the local UUri maps to `upl/`, the remote UUri maps to `upr/`.
-If UAuthority is defined in UUri, it **MUST** be put in the second part.
-Then the rest of the UUri **MUST** be put in the last part.
 
-To be more specific:
+* `up/`: Zenoh key **MUST** always start with `up/` and the scheme (`up://`) in the short UUri is always removed.
```

Comment:
Out of curiosity, is there a reason not to just keep the scheme from the short URI? It's only two extra characters, but removes a whole extra string operation that is unique to Zenoh and applied to a string that is otherwise already formatted correctly. Does Zenoh treat `:` or `/` as special characters?

If there's no specific need for this, we could reduce the amount of code that has to be written, tested, and maintained by keeping the scheme from the short URI.

Reply:
You're right. We can't have `//` in Zenoh key.